x-pack/filebeat/input/awss3/s3_integration_test.go

```diff
@@ -75,7 +77,44 @@ func getConfigForTest(t *testing.T) config {
 	return config
 }
 
-func defaultTestConfig() *common.Config {
+// getConfigForTestBucketCollector function gets aws credentials for integration tests.
+func getConfigForTestBucketCollector(t *testing.T) config {
```

Comment:
This is probably better to be relocated into a common location so other inputs like aws-cloudwatch can also use for testing. Similar to what we have in Metricbeat: https://github.com/elastic/beats/blob/master/x-pack/metricbeat/module/aws/mtest/integration.go#L14

Reply:
the only problem is https://github.com/elastic/beats/blob/master/x-pack/filebeat/input/awss3/config.go#L20
`config` is private and won't be accessible from an eventual `x-pack/filebeat/input/ftest/integration.go`
What do you suggest? I'm not sure making it public is fine